### PR TITLE
Viewing past orders - closes #24

### LINKED ISSUE #24: Viewing Past Orders
Background: An existing user that has multiple orders
      As an Authenticated User
      When I visit "/orders"
      Then I should see all orders belonging to me and no other orders

### PR DESCRIPTION
**Pull request by Laszlo-JFLMTCO**
Implement Orders table, Order model and orders_controller to support feature allowing users to view their own orders.
Locally merged with latest Master. All tests passed.

After merging, everybody will have to run migration again.

@Dpalazzari @drod1000 @njgheorghita

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@
   get '/cart' => 'carts#index'
   post '/cart_increment' => 'carts#increment', :as => 'increment'
   post '/cart_decrement' => 'carts#decrement', :as => 'decrement'
+  get '/orders' => 'orders#index'
```
Comment (njgheorghita) [inline config/routes.rb:14]: create a resource here?

**Comment by Dpalazzari** [comment]
Went through this card with group. Code is top notch 🚀

**Comment by drod1000** [comment]
@Laszlo-JFLMTCO I worked with Laszlo on this. Should be good to merge

**Comment by njgheorghita** [comment]
looks good! i'm gonna merge!

**Pull request merged**